cardinal/enums/enums.go

```diff
@@ -334,3 +334,30 @@ var toEnumTxDef = map[string]TxtDefType{
 	"Object":    TxtDefTypeObject,
 	"Action":    TxtDefTypeAction,
 }
+
+// GrammarType - Custom type for the gramar definition types
+type GrammarType int
+
+const (
+	GrammarTypeNone              GrammarType = iota // Starts at 0
+	GrammarTypeDefinitionArticle                    // 1
+	GrammarTypePreposition                          // 2
+	GrammarTypePrepo                                // 2
+	GrammarTypeAdverb                               // 3
+)
+
+var toStringGrammar = map[GrammarType]string{
```

Comment:
we will keep these but in fact we dont really need them based on the fact that we can parse without looking them up